### ISSUE TITLE
sn.raw.measure does not accept acqTime=0

### ISSUE THREAD

**Issue by antho214** [PicoQuant/snAPI#11]
**Describe the bug**

The `raw.measure` does not accept `acqTime=0`. There is an error printed out in the logging (see below). I have added the complete log with this post. I want to run the measurement for an unknown amount of time. I do not want to start/restart the measurement since this could lead to missing detections. With an `acqTime>0` the measurement works properly.

I have tried with `startBlock` method but this yields similar outcome.

```
# log file
MHlib error in MH_StartMeas (-60): MH_ERROR_MB_ACK_FAIL
```

**To Reproduce**

Below is the python script that I am using.

```python
from snAPI.Main import *
import os 

class MultiHarpController:
    def __init__(self):

        # set the library for your device type
        self.sn = snAPI(libType=LibType.MH)
        self.sn.getDevice()

        self.sn.setLogLevel(LogLevel.Device, False)

        # alternatively read data from file
        self.sn.setLogLevel(LogLevel.DataFile, True)
        self.sn.initDevice(MeasMode.T3)
        
        # set the configuration for your device type
        self.sn.setLogLevel(logLevel=LogLevel.Config, onOff=True)
        basename = os.path.dirname(__file__)
        self.sn.loadIniConfig(os.path.join(basename, "config", "MH.ini"))
        
        # print complete device config structure
        self.sn.logPrint(json.dumps(self.sn.deviceConfig, indent=2))

        # device serial number (name)
        self.sn.logPrint("Serial Number:", self.sn.deviceConfig["ID"])        

    def set_filepath(self, path):
        self.sn.setPTUFilePath(path)

    def get_device_config(self):
        return self.sn.deviceConfig

    def get_num_channels(self):
        return self.sn.getNumAllChannels()
    
    def get_count_rates(self):
        return self.sn.getCountRates()

if(__name__ == "__main__"):

    import time

    controller = MultiHarpController()

    print(controller.get_count_rates())
    controller.sn.raw.measure(
        acqTime=0, size=1024*1024*1024, waitFinished=False, savePTU=True)
    
    time.sleep(2)
    controller.sn.raw.stopMeasure()
```

**Expected behavior**

The documentation mentions that the parameter `acqTime` can be set to 0. I would expect that the measurement runs until the `stopMeasure()` method is called.

**Logfile**

[231220.log](https://github.com/PicoQuant/snAPI/files/13730195/231220.log)
[MH.txt](https://github.com/PicoQuant/snAPI/files/13730251/MH.txt)

**Additional context**

Device: MH160N
Python : 3.10.13
snAPI : 1.0.2

**Comment by tpoint75** [comment]
Hi Anthony,

this error message comes from the device driver (multiharp library) that snAPI uses to communicate with your MH160.
This error means that something goes wrong with the USB connection and should not have anything to do with snAPI directly.

Do you see a red LED on the device? Restart the device and the error should be gone. If not change the USB cable.

Please tell me if it helps and the error message persist!

Note:
snAPI supports a print function to be in sync with the log file:
https://picoquant.github.io/snAPI/snAPI.Main.html#snAPI_Main_snAPI_logPrint

If you don't have any old scripts that explicitly need the raw format, don't use it! The unfold class gives you a more comfortable access to the data you need:
https://picoquant.github.io/snAPI/snAPI.Main.html#snAPI_Main_Unfold

Best regards,
Torsten

**Comment by antho214** [comment]
Hi Torsten,

I have verified the LEDs, the one on the left was orange so I was probably not using a fast enough USB port? I have switched the cable to a USB3.1 port and now all LEDs are green on the MH160. I have restarted the device but the error still persists. I have also tried with a different USB cable with similar results.

**Note.** I have tried both with the `unfold` and `raw` measurement and the error still persist. That being said when `acqTime>0` then everything works as expected.

**Note2.** When I launch the script, the LED on the right becomes red.

```python
controller = MultiHarpController()

controller.get_count_rates()
controller.sn.raw.measure(
    acqTime=0, size=1024*1024*1024, waitFinished=False, savePTU=False)

controller.sn.logPrint("WAITING ...")
time.sleep(2)
controller.sn.logPrint("After sleeping...")
controller.sn.raw.stopMeasure()
controller.sn.logPrint(controller.sn.raw.isFinished())
```

[231221.log](https://github.com/PicoQuant/snAPI/files/13741882/231221.log)
